### PR TITLE
Referencing new MS.VS.Services.Client dll

### LINKED ISSUE #3670: Migrate from referencing MS.VS.Services.Client to MS.VS.Services.Client.Interactive
Tracking internal bug 278033

### PR DESCRIPTION
**Pull request by mishra14**
Fixes: https://github.com/NuGet/Home/issues/3670

Triggered by internal bug 278033. Microsoft.VisualStudio.Services.ClientInteractive.dll has been renamed to Microsoft.VisualStudio.Services.Client.Interactive.dll.

Still waiting on folks from VSCS team to validate that this fixes the issue. But good to have some eyes on this before that.

//cc: @rrelyea @emgarten @joelverhagen @dtivel @drewgil @jainaashish @alpaix @zhili1208 @rohit21agrawal

### CODE REVIEW

File: test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/project.json [hunk 1]
```diff
@@ -3,7 +3,8 @@
     "Moq": "4.2.1507.118",
     "Test.Utility": "4.0.0-*",
     "xunit": "2.1.0",
-    "xunit.runner.visualstudio": "2.1.0"
+    "xunit.runner.visualstudio": "2.1.0",
+    "Newtonsoft.Json": "6.0.0"
```
Comment (joelverhagen) [inline test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/project.json]: This should match the VS version (`6.0.4`)

**Comment by joelverhagen** [comment]
NuGet should start using Newtonsoft.Json 8.0.x for dev15, right?

**Review by joelverhagen — CHANGES_REQUESTED** [review]
Should this PR include the changes to newer Newtonsoft.Json in VS?

**Comment by mishra14** [comment]
@joelverhagen  : Fixed the requested change and created an issue to track the other comments - https://github.com/NuGet/Home/issues/3770

**Review by joelverhagen — APPROVED** [review]

**Comment by mishra14** [comment]
Merged: https://github.com/NuGet/NuGet.Client/commit/072270773bafb3e2ef22178b7b5fc791e23c5891

**Pull request closed without merging**